docs/05-themes-components/01-developing-themes.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Developing Discourse Themes & Theme Components
+short_title: Developing Themes
+id: beginners-guide
+---
+
+Discourse Themes and Theme Components can be used to customize the look, feel and functionality of Discourse's frontend. This section of the developer guides aims to provide all the reference materials you need to develop simple themes for a single site, right up to complex open-source theme components.
+
+This introduction aims to provide a map of all the tools and APIs for theme development. If you prefer a step-by-step tutorial for theme development, jump straight to out [Building your first theme]().
+
+## Themes vs. Theme Components
+
+**Discourse Themes** can be used to customize the frontend user experience using CSS and Javascript. Each theme has its own git repository, and community admins can generally install and manage them via the Discourse admin panel, even on shared hosting platforms.
+
+**Theme Components** are themes which are intended for use alongside other Theme Components, as part of an overall Theme. From a development point of view, Theme Components and Themes are almost identical. In these guides, the phrase "Theme" and "Theme Component" are used interchangeably.
```

Comment:
I don't know maybe this duplication is good, but this is very much a repetition of docs/07-theme-developer-tutorial/01-introduction.md

Reply:
Yeah it's kinda deliberate. I think some people prefer 'reference guides' like this one, but others will jump straight to a step-by-step tutorial